assets/src/js/components/Modal.js

```diff
@@ -25,12 +32,22 @@ var Modal = React.createClass( {
 		}
 	},
 
+	onErrorClick: function () {
+		this.setState( { showErrors: !this.state.showErrors } );
+	},
+
 	render : function () {
 		var classes        = 'locomotive-overlay',
-		    batchInfo     = this.props.batchInfo,
-		    batchTitle    = ( batchInfo.batchTitle ) ? batchInfo.batchTitle : this.props.selectedBatch,
-		    status         = ( batchInfo.status ) ? batchInfo.status : '',
-		    progressStyle = { width: batchInfo.progress + '%' };
+			batchInfo     = this.props.batchInfo,
```

Comment:
In GH this area looks like the spacing is different for each line- not sure if thats just a GH thing or if it's aligned properly in the text editor. 🌴